cairo/src/context.rs

```diff
@@ -815,9 +815,9 @@ mod tests {
     }
 
     #[test]
-    fn clip_rectange() {
+    fn clip_rectangle() {
         let ctx = create_ctx();
-        let rect = ctx.copy_clip_rectangle_list();
+        let rect = ctx.copy_clip_rectangle_list()?;
```

Comment:
Since the function doesn't return a `Result`, I think it'll fail. Better use a `expect` here I think.

Reply:
Indeed that did not even compile, I forgot to amend

Reply:
> Please remove the update on the `gir` submodule too. :)

I wonder how many silly mistakes I can still make in a row :(
Hopefully ok now